swarm/storage/mru/resource.go

```diff
@@ -108,9 +108,6 @@ func NewError(code int, s string) error {
 	return r
 }
 
-// Signature is an alias for a static byte array with the size of a signature
-type Signature [signatureLength]byte
-
```

Comment:
This moved to `updaterequest.go`. At the end of the refactor in PR #704, it will end up where signatures actually take place.